framework/src/modules/http_api/index.js

```diff
@@ -50,15 +50,15 @@ class HttpAPIModule extends BaseModule {
 		return {};
 	}
 
-	async load(channel) {
+	load(channel) {
```

Comment:
These `load` and `unload` methods must be kept as `async` for every module as pattern defined by SDK. Look for `BaseModule` class.